### ISSUE TITLE
Question : Should this transition to a static class?

### ISSUE THREAD

**Issue by stewarty** [pbc-web/post-type-as-taxonomy#2]
Question : Should this transition to a static class?

As there doesn't seem to be anytime when this class gets any properly modified, should it we transition it to being static ?

**Comment by leanderiversen** [comment]
Yes, I believe the class should transition to being a static class, as I cannot see any scenarios where it needs modification.
